verilog/tools/syntax/verilog_syntax_test.sh

```diff
@@ -34,6 +34,34 @@ function strip_error() {
   sed -e 's| (syntax-error).||'
 }
 
+# Takes some json input from stdin and creates a string that has all
+# dictionaries sorted and all whitespace canonicalized.
+# Very simplistic, and output is not necessarily parseable as json, but goal
+# is to have some comparable string (will break if comma in strings)
+function json_canonicalize() {
```

Comment:
Could be in its own standalone shell script?  Seems very useful.

Reply:
I guess once it is used in more than one place, we can move it out.